src/Svg/Tag/Ellipse.php

```diff
@@ -2,7 +2,6 @@
 /**
  * @package php-svg-lib
  * @link    http://github.com/PhenX/php-svg-lib
```

Comment:
this line and package could go away ?

Reply:
Or update it with the correct location? I want to minimize file headers, but I think the link to the project remains useful (even though it's also present in the composer.json).

Reply:
Maybe update is good yes, package is redundant with the URL :)

Reply:
Updated the URL. Perhaps the package is redundant but it's consistent with other Dompdf projects so I will leave it for now. I'll consider for a future release how we might optimize the file header.